modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AspNetCoreServerCodegen.java

```diff
@@ -375,4 +329,69 @@ protected void processOperation(CodegenOperation operation) {
     public String toRegularExpression(String pattern) {
         return escapeText(pattern);
     }
+
+    private void  setCliOption(CliOption cliOption) throws IllegalArgumentException {
+        if (additionalProperties.containsKey(cliOption.getOpt())) {
+            cliOption.setOptValue(additionalProperties.get(cliOption.getOpt()).toString());
+            if (classModifier.getOptValue() == null) {
```

Comment:
Copy paste error - line should have cliOption.getOptValue not classModifier.getOptvalue